zerver/lib/email_notifications.py

```diff
@@ -733,6 +779,29 @@ def enqueue_welcome_emails(user: UserProfile, realm_creation: bool = False) -> N
     )
     context["getting_user_started_link"] = user.realm.uri + "/help/getting-started-with-zulip"
 
+    # We only send the followup_zulip_guide email for a subset of Realm.ORG_TYPES
+    organization_type = user.realm.org_type
+    send_zulip_guide_followup = organization_type in FOLLOWUP_ZULIP_GUIDE_CONTEXT
+
+    # Only send follow_zulip_guide to "/for/communities" guide if user is realm admin.
+    # TODO: Remove this condition when "/for/communities" is updated.
+    if send_zulip_guide_followup and not user.is_realm_admin:
+        send_zulip_guide_followup = (
+            FOLLOWUP_ZULIP_GUIDE_CONTEXT[organization_type]["for_zulip_link"]
+            != "https://zulip.com/for/communities"
+        )
+
+    if send_zulip_guide_followup:
+        context.update(
+            zulip_guide_subject=str(
```

Comment:
Hmm, I wouldn't have expected that doing the `str()` part here would be correct; I would expect that we want to let the email sending code path resolve these lazy strings from the context into strings inside `build_email`? 

Or maybe have this block be inside a `with override_language(user.default_language)` type of block.

Reply:
Updated changes so that the strings are not sent / stored as context since that cannot be translated in our current system.

Instead the strings are all marked for translation in the `onboarding_zulip_guide` email templates and a string/enum context value is passed so that only the string for the organization's type is rendered and translated in the email.